src/java/picard/util/MathUtil.java

```diff
@@ -164,6 +164,16 @@ public static short min(final short[] nums) {
         return min;
     }
 
+    /** Returns the smallest value stored in the array. */
```

Comment:
@eitanbanks please review this...it snuck in while I was working on something else....or say the word and I'll excise it and submit it with a new PR.